frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
```

Comment:
added because it's present in `eslint-plugin-react-app` from CRA, but it's really minimal stuff and proper accessibility checks through a storybook addon are also on my to-research list.